Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -0,0 +1,43 @@
+# Converts text protos to binary protos and writes the output binary protos to
+# the output folder that is specified in SCRIPT_OUTPUT_FILE_0, which is
+# retrieved from the Output Files of the Run Script Build Phase that executes
+# this script.
+
+# Directory that contains the text protos to convert to binary protos.
+text_protos_dir=${SCRIPT_INPUT_FILE_0}
```

Comment:
Nit: quote variables? Shell style guide encourages quoting. (applies throughout)

Reply:
I did that but I might need you to make sure it is correct. The script executes correctly though.